### PR TITLE
Update size-your-cluster.md

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/size-your-cluster.md [hunk 1]
```diff
@@ -89,11 +89,12 @@ To learn performance test results of different cluster scales, see [TiDB Cloud P
...
-- 8 vCPU, 32 GiB
-- 8 vCPU, 64 GiB
-- 16 vCPU, 64 GiB
-- 32 vCPU, 128 GiB
+| Standard Size | High Memory Size | 
+|:---------:|:----------------:|
+| 4 vCPU, 16 GiB  |          |
+| 8 vCPU, 32 GiB    | 8 vCPU, 64 GiB        |
+| 16 vCPU, 64 GiB  | Coming soon       |
+| 32 vCPU, 128 GiB   |    |
```
Comment (hfxsd) [inline tidb-cloud/size-your-cluster.md:97]:
```suggestion
| 4 vCPU, 16 GiB  |  NA        |
| 8 vCPU, 32 GiB    | 8 vCPU, 64 GiB        |
| 16 vCPU, 64 GiB  | Coming soon       |
| 32 vCPU, 128 GiB   |  NA  |
```

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/size-your-cluster.md [hunk 2]
```diff
@@ -23,12 +23,12 @@ To learn performance test results of different cluster scales, see [TiDB Cloud P
...
 
-- 4 vCPU, 16 GiB
-- 8 vCPU, 16 GiB
-- 8 vCPU, 32 GiB
-- 16 vCPU, 32 GiB
-- 32 vCPU, 64 GiB
-- 32 vCPU, 128 GiB
+| Standard Size | High Memory Size | 
+|:---------:|:----------------:|
+| 4 vCPU, 16 GiB  |          |
```
Comment (qiancai) [inline tidb-cloud/size-your-cluster.md:28]:
```suggestion
| 4 vCPU, 16 GiB  | N/A          |
```

File: tidb-cloud/size-your-cluster.md [hunk 3]
```diff
@@ -23,12 +23,12 @@ To learn performance test results of different cluster scales, see [TiDB Cloud P
 
 The supported vCPU and RAM sizes include the following:
 
-- 4 vCPU, 16 GiB
-- 8 vCPU, 16 GiB
-- 8 vCPU, 32 GiB
-- 16 vCPU, 32 GiB
-- 32 vCPU, 64 GiB
-- 32 vCPU, 128 GiB
+| Standard Size | High Memory Size |
```
Comment (qiancai) [inline tidb-cloud/size-your-cluster.md:26]:
```suggestion
| Standard size | High memory size | 
```

File: tidb-cloud/size-your-cluster.md [hunk 4]
```diff
@@ -89,11 +89,12 @@ To learn performance test results of different cluster scales, see [TiDB Cloud P
 
 The supported vCPU and RAM sizes include the following:
 
-- 4 vCPU, 16 GiB
-- 8 vCPU, 32 GiB
-- 8 vCPU, 64 GiB
-- 16 vCPU, 64 GiB
-- 32 vCPU, 128 GiB
+| Standard Size | High Memory Size |
```
Comment (qiancai) [inline tidb-cloud/size-your-cluster.md:92]:
```suggestion
| Standard size | High memory Size | 
```

File: tidb-cloud/size-your-cluster.md [hunk 5]
```diff
@@ -89,11 +89,12 @@ To learn performance test results of different cluster scales, see [TiDB Cloud P
...
-- 8 vCPU, 32 GiB
-- 8 vCPU, 64 GiB
-- 16 vCPU, 64 GiB
-- 32 vCPU, 128 GiB
+| Standard Size | High Memory Size | 
+|:---------:|:----------------:|
+| 4 vCPU, 16 GiB  |  NA        |
+| 8 vCPU, 32 GiB    | 8 vCPU, 64 GiB        |
+| 16 vCPU, 64 GiB  | Coming soon       |
+| 32 vCPU, 128 GiB   |  NA  |
```
Comment (qiancai) [inline tidb-cloud/size-your-cluster.md:97]:
```suggestion
| 4 vCPU, 16 GiB  |  N/A        |
| 8 vCPU, 32 GiB    | 8 vCPU, 64 GiB        |
| 16 vCPU, 64 GiB  | Coming soon       |
| 32 vCPU, 128 GiB   |  N/A  |
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: tidb-cloud/size-your-cluster.md (hunk 5 above)
Comment (qiancai) [reply tidb-cloud/size-your-cluster.md:97]: In some contexts, "NA" might be confused with "North America"

**Pull request merged**